pfcpiface/parse-far.go

```diff
@@ -78,9 +83,11 @@ func (f *far) parseFAR(farIE *ie.IE, fseid uint64, upf *upf, op operation) error
 		return err
 	}
 
+	var flag bool = false
```

Comment:
bad name. Instead can we use bit mask to set what IEs are present and `&` with `const` bitmasks representing mandatory IEs for create/update?

Reply:
enum in golang https://yourbasic.org/golang/iota/

Reply:
done